src/quo2/components/list_items/dapp/style.cljs

```diff
@@ -0,0 +1,44 @@
+(ns quo2.components.list-items.dapp.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(defn container-outer
+  [state blur? theme]
+  {:flex               1
+   :padding-horizontal 12
+   :padding-vertical   8
+   :border-radius      12
+   :background-color   (cond
+                         (= state :default)
+                         :transparent
+
+                         (and (= state :pressed) (= theme :dark) blur?)
+                         colors/white-opa-5
+
+                         (= state :pressed)
+                         (colors/custom-color :blue 50 5)
```

Comment:
We should not use :blue here, we want to pass 'customization-color' as a prop. You can check many other components that have that

Reply:
Yes, thank you. done.

Reply:
@J-Son89 does the current implementation (using `customization-color`) look ok?

Reply:
ok, noted your comment below. done